web/navigation-v3/index.html

```diff
@@ -183,67 +206,85 @@ <h2 class="visuallyhidden">Main</h2>
 								<a href="page.html" class="nav-link">Business Ecosystems</a>
 								<ul class="clean-list" role="list">
 										<li>
-											<a href="page.html" class="nav-link">Automotive and transportation</a>
+											<a href="page.html" class="nav-link">eCommerce</a>
 										</li>
 										<li>
 											<a href="page.html" class="nav-link">Media and entertainment</a>
 										</li>
+										<li>
+											<a href="page.html" class="nav-link">Network and communications</a>
+										</li>
 										<li>
 											<a href="page.html" class="nav-link">Publishing</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Web payments</a>
+											<a href="page.html" class="nav-link">Security and privacy</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Web of Data</a>
+											<a href="page.html" class="nav-link">Smart cities</a>
+										</li>
+										<li>
+											<a href="page.html" class="nav-link">Transportation</a>
+										</li>
+										<li>
+											<a href="page.html" class="nav-link">Web advertising</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Web and telecommunications</a>
+											<a href="page.html" class="nav-link">Web of Data</a>
 										</li>
 										<li>
 											<a href="page.html" class="nav-link">Web of Things</a>
 										</li>
 									</ul>
 							</li>
+							<li>
+								<a href="page.html" class="nav-link">Liaisons</a>
+							</li>
 							<li class="has-children">
-								<a href="page.html" class="nav-link">Supporting W3C</a>
+								<a href="page.html" class="nav-link">Support W3C</a>
 								<ul class="clean-list" role="list">
 										<li>
-											<a href="page.html" class="nav-link">Donate now</a>
+											<a href="page.html" class="nav-link">Donate</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Global sponsorship</a>
-										</li>
-										<li>
-											<a href="page.html" class="nav-link">More ways to support us</a>
+											<a href="page.html" class="nav-link">Sponsorship</a>
 										</li>
 									</ul>
 							</li>
 							<li class="has-children">
 								<a href="page.html" class="nav-link">W3C Membership</a>
 								<ul class="clean-list" role="list">
 										<li>
-											<a href="page.html" class="nav-link">Membership home</a>
+											<a href="page.html" class="nav-link">Member home</a>
 										</li>
 										<li>
 											<a href="page.html" class="nav-link">Membership fees</a>
 										</li>
 										<li>
 											<a href="page.html" class="nav-link">Join W3C</a>
 										</li>
+										<li>
+											<a href="page.html" class="nav-link">Our members</a>
+										</li>
+										<li>
+											<a href="page.html" class="nav-link">Membership testimonials</a>
+										</li>
```

Comment:
Membership testimonials should be removed (as this page is going to be merged with the above "Our members" list)

```suggestion
```

Reply:
My goal from this PR was to make the menus match what's in the site map before asking for review by the team. If some of these things have already been decided, shouldn't they be updated in the site map doc? Maybe it would be worthwhile for us to do a quick review and edit of the site map with Coralie (tomorrow, Jan 21)? I am not really clear on the intended change control for the site map doc.